deeplake/core/io.py

```diff
@@ -534,8 +549,11 @@ def _map_chunk_engines(self, tensors: Sequence[str]) -> Dict[str, ChunkEngine]:
 
     def _create_chunk_engine(self, tensor_name, version_state):
         tensor_key = version_state["tensor_names"][tensor_name]
+        tensor_meta = version_state["full_tensors"][tensor_key].meta
+        cache_size = max(self.cache_size, tensor_meta.max_chunk_size or 0)
```

Comment:
why do we compare it with 0? is it related to case when tensor_meta.max_chunk_size is None?

Reply:
yes